src/MaaWpfGui/Res/Localizations/en-us.xaml

```diff
@@ -846,15 +846,15 @@ The video aspect ratio needs to be 16:9 without interference factors such as bla
 
 The support for reclamation algorithm is still under development at present. Please mind the following:
 
-In normal mode, MAA: 
+Without save file: farm points through initial setup.
   1. Cannot be used when a save already exists. (simply delete the current save)
   2. Cannot be used when squads are not empty. (simply clear the current squad)
-  3. Can only be started at the main nav page of reclamation algorithm. 
+  3. Can only be started at the main nav page of reclamation algorithm.
 
-To get points by manufacturing: 
-  1. it must be the first day after settlement, and there should be no enemy attacks entering the stationed area for the next three days. 
+With save file: farm points through crafting (advanced).
+  1. It must be the first day after settlement, and there should be no enemy attacks entering the stationed area for the next three days.
   2. It must start when entering the reclamation algorithm. (stationed location needs to be visible)
-  3. If the quantity that can be manufactured is exactly a multiple of 99, MAA can be confused (stuck). You can use up a little before saving. This might be repaired later. 
+  3. If the quantity that can be manufactured is exactly a multiple of 99, MAA can get confused (stuck). You can use up a little before saving. This might be repaired later.
```

Comment:
@ABA2396 
1. By settlement we mean first day after "saving the file"?
2. Must start when entering RA? Stationed location visible? What's that?

Reply:
1. The first day after obtaining points (either completing a 3-day settlement or loading a saved file).
2. A place where you can see your base, which is within the map, different from the start screen without a saved file.